app/src/main/res/menu/navigation_drawer_menu.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  tools:showIn="navigation_view">
+  <group android:checkableBehavior="single">
+    <item
+      android:id="@+id/nav_home"
+      android:icon="@drawable/ic_home_icon_grey_24dp"
+      android:title="@string/menu_home" />
+    <item
+      android:id="@+id/nav_help"
+      android:icon="@drawable/ic_help_grey_icon_24dp"
+      android:title="@string/menu_help"
+      android:visible="true" />
```

Comment:
Why is `nav_help` visible and in `nav_home` it has not been defined?

Reply:
Removed